packages/flutter/test/cupertino/date_picker_test.dart

```diff
@@ -253,9 +253,9 @@ void main() {
       ),
     );
 
-    await tester.drag(find.text('40'), _kRowOffset);
+    await tester.drag(find.text('40'), _kRowOffset, warnIfMissed: false);
```

Comment:
Why is it ok to miss here?

Reply:
I guess, I have that question for most of the tests where missing is ok...

Reply:
The way the Cupertino pickers are built, the text isn't what actually gets hit at the location where the text renders. (The actual construction is non-trivial and I'm not really sure I could accurately describe what's being hit exactly. I spent some time trying to debug it but there's just too many nodes there for me to figure it out.)

Reply:
added a note to this file